lib/config/migrations/custom/unpublish-safe-migration.spec.ts

```diff
@@ -83,12 +83,26 @@ describe('config/migrations/custom/unpublish-safe-migration', () => {
   it('prevent duplicates', async () => {
     await expect(UnpublishSafeMigration).toMigrate(
       {
-        extends: ['npm:unpublishSafe'],
+        extends: ['security:minimumReleaseAgeNpm'],
         unpublishSafe: true,
       },
+      {
+        extends: ['security:minimumReleaseAgeNpm'],
+      },
+    );
+  });
+
+  it('should not migrate npm:unpublishSafe', async () => {
+    // NOTE that this preset name is now deprecated, but will be handled by `ExtendsMigration` instead of handled in this migration
+
+    await expect(UnpublishSafeMigration).toMigrate(
+      {
+        extends: ['npm:unpublishSafe'],
+      },
       {
         extends: ['npm:unpublishSafe'],
       },
+      false,
```

Comment:
Is this the right way to do this?

Reply:
I don't really know / remember this migration. 😅